ci/code_checks.sh

```diff
@@ -73,9 +73,30 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
         -i "pandas.Period.freq GL08" \
         -i "pandas.Period.ordinal GL08" \
         -i "pandas.RangeIndex.from_range PR01,SA01" \
+        -i "pandas.Series.cat.add_categories PR01,PR02" \
```

Comment:
It looks like merge conflict-resolution went the wrong way here. I think these should likely all be removed.